src/mlpack/methods/ann/ffn_impl.hpp

```diff
@@ -84,12 +84,13 @@ void FFN<OutputLayerType, InitializationRuleType, CustomLayers...>::Train(
 
   Log::Info << "FFN::FFN(): final objective of trained model is " << out
       << "." << std::endl;
+  return out;
```

Comment:
Might be more intuitive to call this loss, or objective, what do you think?

Reply:
Yes, correct but it was already declared or defined. So I thought it's better not to change it's name. Also at every optimizer function it was written like that so I decided to keep it as it is.

Reply:
I see, I don't mind to leave it like this.